### PR TITLE
Paragraph translation sanity check

### PR DESCRIPTION
**Pull request by AronNovak**

### CODE REVIEW

File: web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+
+/**
+ * Tests entity references.
+ */
+class ServerGeneralParagraphTranslatability extends ExistingSiteBase {
+
+  /**
+   * Checks all entity reference fields to nodes use the select2 widget.
```
Comment (AronNovak) [inline web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php:13]:
```suggestion
   * Checks all entity reference revision fields for Paragraphs are non-translatable.
```

File: web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+/**
+ * Tests entity references.
+ */
+class ServerGeneralParagraphTranslatability extends ExistingSiteBase {
+
+  /**
+   * Checks all entity reference revision fields for Paragraphs are non-translatable.
+   */
+  public function testParagraphsTranslation() {
```
Comment (amitaibu) [inline web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php:15]: Another idea that might be better - can we throw an Exception if one tries to set the field incorrectly?

Comment (amitaibu) [reply web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php:15]: Good point :+1: Save ourselves from ourselves :)

File: web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        continue;
+      }
+
+      $storage_definition = $field_config->getFieldStorageDefinition();
+      $referencing = $storage_definition->getSetting('target_type');
+      if ($referencing != 'paragraph') {
+        continue;
+      }
+
+      $this->assertFalse($field_config->isTranslatable(), sprintf("%s must not be translatable", $id));
```
Comment (amitaibu) [inline web/modules/custom/server_general/tests/src/ExistingSite/ServerGeneralParagraphTranslatability.php:30]:
```suggestion
      $this->assertFalse($field_config->isTranslatable(), sprintf("%s must not be translatable as it is a Paragraph field", $id));
```

**Review by amitaibu — APPROVED** [review]

**Pull request merged**